### ISSUE TITLE
Automatically change the bounds on the PMT plot applet

### ISSUE THREAD

**Issue by sbuechele** [Jayich-Lab/jax#70]
The PMT plotting applet has the ability to automatically update its bounds, however when we are running experiments, no PMT data is written. Therefore, the bounds do not update when it jumps back to collecting data.

**Comment by fanmingyu212** [comment]
This bug is related to how `RealtimePlotApplet` is designed. It is designed to automatically update the bounds when the latest data point is within the current bounds, and does not update when the latest data point is beyond the current bounds. This prevents the graph jumping to the latest data point when the user wants to take a look at the previous data.

To solve the bug, the applet needs to know whether the user is looking at previous data or not. If the user is looking at previous data, it should not move the bounds, otherwise it should move.

**Comment by sbuechele** [comment]
Agreed, although I would propose just a listener that triggers a bypass of "not updating when the latest data point is beyond the current bounds", whenever there is an end to an experiment
